packages/compass-e2e-tests/tests/global-preferences.test.ts

```diff
@@ -177,6 +177,9 @@ describe('Global preferences', function () {
     const compass = await init(this.test?.fullTitle());
     const browser = compass.browser;
     try {
+      // TODO: check that the shell is there before toggling the section so we
```

Comment:
We need a follow-up ticket for once we hide the shell entry points when the shell is disabled.